PHP/PHP Source.sublime-syntax

```diff
@@ -643,7 +643,7 @@ contexts:
       scope: punctuation.separator.php
     - match: '\?'
       scope: storage.type.nullable.php
-    - match: \b(array|bool|int|float|string)\b
+    - match: \b(array|bool|int|float|string|object)\b
```

Comment:
- add the missing `callable` as well?
  - make these 2 changed lines exactly the same.
    - make the following into a new context so we don't have to change this everywhere in the future?

```yaml
- match: \b(array|callable|int|string|bool|float|object)\b
  scope: storage.type.php
```